cycle3/Boomtown/Preregistration/README.md

```diff
@@ -1,39 +1,70 @@
 # NGS2 Docker Cycle 3
 
-**Most up-to-date:**
-
-As of August 02, 2019 - https://github.com/GallupGovt/ngs2/edit/master/cycle3/Boomtown/Preregistration
-
 [Pablo Diego-Rosell](https://github.gallup.com/Pablo-Diego-Rosell "Gallup GitHub")
+
 [Nasser Qadri](https://github.gallup.com/nasser-qadri "Gallup GitHub")
+
 [Taylor Bolt](https://github.gallup.com/taylor-bolt "Gallup GitHub")
+
 [Mitchell Bregman](https://github.gallup.com/mitchell-bregman "Gallup GitHub")
 
-## To connect to AWS instance in terminal use: 
-`ssh -i .ssh/id_rsa {first_last}@dc-dev-docker.gallupaws.com`
+-----
+
+## Prerequisites:
+
+- Must have Docker installed
```

Comment:
Can you please add links to the relevant sites? Ideally the link will point to a permanent repository with a specific version of each software package, to avoid dependency incompatibility issues.